deno_core looks like this:

    [features]
    default = [
        "include_icu_data",
        "v8_use_custom_libcxx",
    ]
    include_icu_data = ["deno_core_icudata"]
    v8_use_custom_libcxx = ["v8/use_custom_libcxx"]

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -43 +43 @@
-deno_core = { version = "0.280.0" }
+deno_core = { version = "0.280.0", default-features = false }
